ui/app/models/transform.js

```diff
@@ -83,12 +86,49 @@ export default Model.extend({
     subText: 'Search for an existing role, type a new role to create it, or use a wildcard (*).',
     wildcardLabel: 'role',
   }),
+  deletion_allowed: attr('boolean', {
```

Comment:
Wrote these params based on the docs - so a second set of eyes checking them for typos/accuracy would be helpful!